sdk/appconfiguration/app-configuration/test/index.readonlytests.spec.ts

```diff
@@ -50,10 +50,33 @@ describe("AppConfigurationClient (set|clear)ReadOnly", () => {
     );
     await assertThrowsRestError(
       () =>
-        client.deleteConfigurationSetting({ key: testConfigSetting.key, label: testConfigSetting.label
+        client.deleteConfigurationSetting({
+          key: testConfigSetting.key,
+          label: testConfigSetting.label
         }),
       409,
       "Delete should fail because the setting is read-only"
     );
   });
+
+  it("accepts operation options", async () => {
+    let storedSetting = await client.getConfigurationSetting({
+      key: testConfigSetting.key,
+      label: testConfigSetting.label
+    });
+
+    try {
+      await client.setReadOnly(testConfigSetting, { requestOptions: { timeout: 1 } });
+      assert.fail("setReadOnly request should abort");
+    } catch (e) {
+      assert.equal(e.code, "REQUEST_ABORTED_ERROR");
```

Comment:
You might need to rebase... core-http now throws AbortError instead of a RestError when an operation is cancelled.

Reply:
Indeed, fixed now.